src/heads-up/App.vue

```diff
@@ -18,14 +18,18 @@
 </template>
 
 <script>
-  import { mapState } from 'vuex'
+  import { mapState } from 'vuex';
   import { AppSidebar, ExternalLink, RefreshButton } from './components'
 
   export default {
     name: 'App',
     components: { AppSidebar, ExternalLink, RefreshButton },
     computed: {
       ...mapState(['head']),
+      ...mapState(['theme']),
+      themeClass() {
+        return this.theme === 'dark' ? '-theme-with-dark-background' : ''
```

Comment:
Maybe add a comment and a link to the source code of chrome devtools to make it understandable why this class name is different from our BEM naming convention in this project.

Reply:
added